projects/elife.yaml

```diff
@@ -437,6 +439,10 @@ journal:
                 engine: redis
                 type: cache.t2.micro
                 clusters: 2
+                overrides:
+                    2:
+                        configuration:
```

Comment:
We only create a single `AWS::ElastiCache::ParameterGroup` for all clusters in the template at the moment, which means overrides do not work for `configuration`: https://github.com/elifesciences/builder/blob/master/src/buildercore/trop.py#L779-L785

Reply:
The build detected it:
`AssertionError: `configuration` override is not allowed for single elasticache clusters`